src/components/header/PageHeader.jsx

```diff
@@ -64,6 +64,15 @@ class PageHeader extends Component {
     filterRedemptions: () => { },
   };
 
+  constructor(props) {
```

Comment:
The constructor comes after static properties/methods, it shouldn't have been moved